docker-compose-travis.yml

```diff
@@ -42,7 +40,12 @@ services:
       test: service nginx status
     volumes:
       - ./clerk-frontend/nginx/certs:/etc/nginx/certs:ro
+
   mock-nlx:
     build: mock-nlx/
     ports:
       - 80:80
+
+networks:
+  default:
+    name: waardepapieren
```

Comment:
The network will be created automatically as part of the `docker-compose up` command. However I will still add it to the README since the network must be created before using the `docker-compose-ipv8.yml`